frappe/utils/background_jobs.py

```diff
@@ -418,6 +420,9 @@ def start_worker_pool(
 
 	WARNING: This feature is considered "EXPERIMENTAL".
 	"""
+	# Set multiprocessing start method to fork instead of forkserver
```

Comment:
Can't seem to reproduce this issue with `rq` itself, only in frappe env.

`test_multi_queue_burst_consumption_worker_pool` breaks basically, jobs just don't run.

Reply:
Also unsure if this is related?

```
13:21:07 web.1         | /home/akhil/.local/share/uv/python/cpython-3.14.0-linux-x86_64-gnu/lib/python3.14/multiprocessing/resource_tracker.py:297: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown: {'/mp-2ci038cs'}
```

Reply:
Works fine with the no-fork worker, i.e. `FRAPPE_BACKGROUND_WORKERS_NOFORK=true`

Reply:
Made the `import multiprocessing` conditional - resolved the leaked semaphore warning.